app/views/providers/proceeding_loop/emergency_defaults/show.html.erb

```diff
@@ -6,19 +6,22 @@
     ) do |form| %>
   <%= page_template template: :basic, form: form do %>
     <span class="govuk-caption-l"><%= position_in_array(@proceeding) %></span>
-    <h1 class="govuk-heading-xl"><%= @proceeding.meaning %></h1>
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-4"><%= @proceeding.meaning %></h1>
+
+    <h2 class="govuk-heading-m"><%= t('.h2_header') %></h2>
+
+    <div class="govuk-!-padding-1"></div>
```

Comment:
Would be much cleaner to add the spacing class name `govuk-!-padding-1` to `h2` and remove this empty element, that only purpose is to apply spacing?